tests/base/src/test/java/org/wildfly/security/sasl/gssapi/TestKDC.java

```diff
@@ -186,7 +181,11 @@ private void stopKDC() {
             return;
         }
 
-        kdcServer.stop();
+        try {
+            kdcServer.stop();
+        } catch (KrbException e) {
+            // ignore
```

Comment:
Might be good to throw an IllegalStateException here to be consistent with other exceptions that are caught in this test class.

Reply:
done